ample/util/tests/test_ample_util.py

```diff
@@ -42,18 +42,42 @@ def test_extract_and_validate_models(self):
 
     def test_extract_and_validate_models_quark(self):
         temp_dir = tempfile.mkdtemp(dir=os.getcwd())
-        amoptd = {'models': os.path.join(self.testfiles_dir, 'result.tar.bz2'),
-                  'models_dir': temp_dir}
+        amoptd = {'models' : os.path.join(self.testfiles_dir, 'result.tar.bz2'),
+                  'models_dir' : temp_dir,
+                  'work_dir' : '.'}
         sequence = 'QPRRKLCILHRNPGRCYDKIPAFYYNQKKKQCERFDWSGCGGNSNRFKTIEECRRTCIG'
         models = ample_util.extract_and_validate_models(amoptd=amoptd, sequence=sequence, single=True)
         self.assertTrue(amoptd['quark_models'])
         self.assertEqual(len(models), 200)
         shutil.rmtree(temp_dir)
+        
+    def test_extract_and_validate_models_quark_zip(self):
```

Comment:
This is probably not necessary but you could avoid adding new test files by simply taking the alldecoy.pdb file and tarring it/zipping it in certain ways before testing. This way you would be able to add tests for all the different types of tarring. Additionally you might want to add tests here to make sure that the function doesn't work with incorrect file types.

Reply:
Agreed, but one for a later commit.